Zenodo.js

```diff
@@ -114,38 +116,79 @@ function doWeb(doc, url) {
 	}
 }
 
+var zoteroType = {
+	"figure": "artwork",
+	 "article": "report"
+}
```

Comment:
I would suggest to move this to the place below where it is actually used. Otherwise I will have forgotten what this is when reading the code below...